pkg/scheduler/profile/profile_test.go

```diff
@@ -102,7 +102,7 @@ func TestNewProfile(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			c := fake.NewSimpleClientset()
 			b := events.NewBroadcaster(&events.EventSinkImpl{Interface: c.EventsV1()})
-			p, err := NewProfile(tc.cfg, fakeFrameworkFactory, NewRecorderFactory(b))
+			p, err := newProfile(tc.cfg, fakeFrameworkFactory, NewRecorderFactory(b))
```

Comment:
Not much point on testing `newProfile` anymore, as it is an unexported function.

Could you migrate any relevant tests to `TestNewMap` or `runtime.TestNewFramework`? The last one doesn't really exist, but there is a related one `runtime.TestNewFrameworkPluginDefaults`